### PR TITLE
Add Delegate page

### LINKED ISSUE #5: Initial Version of Delegation / Voting Weight Page
The initial version of the "Delegate" tab should include:

- [x] Show your L2 token balance
- [x] Show who your L2 tokens are currently delegated to (will default to yourself)
- [x] Show your current total voting weight (i.e. delegated power to your address)
- [x] Enable the user to set their delegate

Possible stretch:

- [ ] In a sub-section of the UI, also show L1 token balance / voting weight and link to other platform(s) to enable setting/updating this

### PR DESCRIPTION
**Pull request by alexkeating**
#### Description

- UI to handle the delegate page
- I also implemented some basic functionality, but am having trouble testing it until we sort out what is going on with our contract deployments.

Closes #5

### CODE REVIEW

**Comment by alexkeating** [comment]
Current dependencies on/for this PR:
* main
  * **PR #9** <a href="https://app.graphite.dev/github/pr/ScopeLift/l2-flexible-voting-frontend/9" target="_blank">Graphite</a> 
    * **PR #18** <a href="https://app.graphite.dev/github/pr/ScopeLift/l2-flexible-voting-frontend/18" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ScopeLift/l2-flexible-voting-frontend/18?utm_source=stack-comment).

File: abi/WormholeL2ERC20.sol/WormholeL2ERC20.ts [hunk 1]
```diff
@@ -0,0 +1,6006 @@
+export default {
+  "abi": [
```
Comment (wildmolasses) [inline abi/WormholeL2ERC20.sol/WormholeL2ERC20.ts:2]: i'm fine seeing this; we can move to human readable in follow-up if helpful

File: components/ConnectWallet.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  action: string;
+  isConnected: boolean;
+  children: ReactNode;
+  className: string;
+};
+
+const ConnectWallet = ({ action, isConnected, children, className }: Props) => {
+  const connectContainer = (
+    <div className={`flex flex-col gap-2 h-full w-full justify-center align-center ${className}`}>
+      <h1 className="self-center text-3xl">Connect wallet to {action}.</h1>
```
Comment (wildmolasses) [inline components/ConnectWallet.tsx:14]: nice one! will support #32

**Review by wildmolasses — APPROVED** [review]
just a couple comments, lgtm

**Comment by wildmolasses** [comment]
### Merge activity

* **Oct 24, 12:13 PM**: @wildmolasses started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/ScopeLift/l2-flexible-voting-frontend/18).

**Pull request merged**